Y26-214 - [BUG] unknown state for child tube causing 500 server error

**Describe the bug**
unknown state for child tube causing 500 server error

**FreshSerivce/RT Ticket Number**
#SR-70229

**To Reproduce**
Steps to reproduce the behaviour:
In Limber prod, visit the plate main page:  SQPP-103563-P
2. Relative section, click on the first (or any) tube child 
3. 500 server error

**Expected behaviour**
logs from the server 

```
   ActionView::Template::Error ("unknown" is not a known state value)
        Caused by: ArgumentError ("unknown" is not a known state value)
        
        Information for: ActionView::Template::Error ("unknown" is not a known state value):
            21: <% end %>
            22: 
            23: <%= card title: 'Suggested Actions', css_class: 'suggested-actions' do %>
            24:   <% presenter.control_suggested_library_passing do %>
            25:     <%= render partial: 'work_completion_form', locals: { presenter: } %>
            26:   <% end %>
            27:   <% presenter.control_additional_creation do -%>
        
        app/views/tubes/sidebars/_default.html.erb:24
        app/helpers/page_helper.rb:36:in 'block in PageHelper#card'
        app/helpers/page_helper.rb:31:in 'PageHelper#card'
        app/views/tubes/sidebars/_default.html.erb:23
        app/views/tubes/sidebars/_container.html.erb:3
        app/helpers/page_helper.rb:27:in 'PageHelper#sidebar'
        app/views/tubes/sidebars/_container.html.erb:1
        app/views/tubes/show.html.erb:15
        app/views/tubes/show.html.erb:14
        app/controllers/labware_controller.rb:25:in 'block (2 levels) in LabwareController#show'
        app/controllers/labware_controller.rb:24:in 'LabwareController#show'
```

--- a/app/integrations/sequencescape/api/v2/base.rb
+++ b/app/integrations/sequencescape/api/v2/base.rb
@@ -23,6 +23,11 @@ class Sequencescape::Api::V2::Base < JsonApiClient::Resource # rubocop:todo Styl
 
   # set the api base url in an abstract base class
   self.site = Limber::Application.config.api.v2.connection_options.url
+
+  connection(true) do |conn| # connection(true) rebuilds the connection
+    conn.faraday.options[:read_timeout] = 300
+  end
+
   api_key = Limber::Application.config.api.v2.connection_options.authorisation
   connection.faraday.headers['X-Sequencescape-Client-Id'] = api_key
 

